airbyte-webapp/src/views/Connection/CatalogTree/CatalogTreeRows.tsx

```diff
@@ -0,0 +1,59 @@
+import { Field, FieldProps, setIn, useFormikContext } from "formik";
+import React, { useCallback } from "react";
+
+import { SyncSchemaStream } from "core/domain/catalog";
+import { AirbyteStreamConfiguration } from "core/request/AirbyteClient";
+import { useConnectionFormService } from "hooks/services/ConnectionForm/ConnectionFormService";
+import { ConnectionFormValues, FormikConnectionFormValues } from "views/Connection/ConnectionForm/formConfig";
+
+import { CatalogSection } from "./CatalogSection";
+import styles from "./CatalogTreeRows.module.scss";
+
+interface CatalogTreeRowsProps {
+  streams: SyncSchemaStream[];
+  onStreamChanged: (stream: SyncSchemaStream) => void;
+}
+
+export const CatalogTreeRows: React.FC<CatalogTreeRowsProps> = ({ streams, onStreamChanged }) => {
```

Comment:
This is is what CatalogTree used to be. Now CatalogTree contains all the headers, bulk edit, and search bar.